packages/smooth_app/lib/pages/product/ocr_ingredients_helper.dart

```diff
@@ -9,11 +9,10 @@ class OcrIngredientsHelper extends OcrHelper {
   String getText(final Product product) => product.ingredientsText ?? '';
 
   @override
-  Product getMinimalistProduct(final Product product, final String text) =>
-      Product(
-        barcode: product.barcode,
-        ingredientsText: text,
-      );
+  Product getMinimalistProduct(final Product product, final String text) {
```

Comment:
The other `getMinimalistProduct` (in `OcrPackagingHelper`) has a slightly different behavior, and that's very unfortunate. It should be exactly the same. Please have a look at it, and check that whatever you change has no impact on the other classes.